.golangci.yml

```diff
@@ -73,21 +76,22 @@ linters:
     # - dupl
     # - errcheck
     # - funlen
+    # - forbidigo
```

Comment:
Looks weird to add commented line. But I understand the goal is to make it similar to stackrox